style.css

```diff
@@ -137,4 +136,88 @@ nav ul {
     fill: var(--gray-2);
     top: 50%;
     transform: translateY(-50%);
+}
+
+/* main */
+
+main {
+    width: 60%;
+}
+
+h1 {
+    font-size: 25px;
+    font-weight: normal;
+}
+
+.filter-question .btn {
+    border-radius: 0;
+}
+
+.filter-question .btn:first-child {
+    border-bottom-left-radius: 0.2em;
+    border-top-left-radius: 0.2em;
+}
+
+.filter-question .btn:nth-last-child(2) {
+    border-bottom-right-radius: 0.2em;
+    border-top-right-radius: 0.2em;
+}
+
+.btn-icon {
+    display: flex;
+    align-items: center;
+    gap: 0.2em;
+}
+
+.btn-icon svg {
+    height: 1em;
+}
+
+.btn-icon.gray svg {
+    fill: var(--gray-2);
+}
+
+.btn-icon.navy svg {
+    fill: var(--navy);
+}
+
+.summary {
+    width: 10%;
+    color: var(--gray-2);
+    flex-shrink: 0;
+    font-size: 14px;
+    padding-left: 3em;
+}
+
+h2 {
+    font-size: 20px;
+    font-weight: normal;
+    color: var(--blue);
+    margin: 0;
+}
+
+.summary p:first-child {
+    margin: 0;
+}
+
+h2 a:visited {
+    color: var(--dark-blue);
+}
+
+.tag-button {
```

Comment:
Can't we delete this and instead use `btn btn-lightblue`? And if we want to remve the border, have just a border rule here.

```
.article-tag {
  border: 0;
}
```